components/Cards/index.js

```diff
@@ -17,3 +17,141 @@
+//   imgDiv.classList.add("img-container");
+
+//   // Content
+//   headlineDiv.textContent = headline;
+//   imgInDiv.src = imgUrl;
+//   span.textContent = authorsName;
+
+//   // Append
+//   cardDiv.appendChild(headlineDiv);
+//   cardDiv.appendChild(authorDiv);
+//   authorDiv.appendChild(imgDiv);
+//   authorDiv.appendChild(span);
+//   imgDiv.appendChild(imgInDiv);
+
+//   return cardDiv;
+// }
+
+// axios
+//   .get("https://lambda-times-backend.herokuapp.com/articles")
+//   .then(response => {
+//     console.log(response);
+//     response.data.articles["javascript"].forEach(item => {
+//       const articleCard = Cards(
+//         item.headline,
+//         item.authorPhoto,
+//         item.authorName
+//       );
+//       articleCards.appendChild(articleCard);
+//     });
+//     response.data.articles["bootstrap"].forEach(item => {
+//       const articleCard = Cards(
+//         item.headline,
+//         item.authorPhoto,
+//         item.authorName
+//       );
+//       articleCards.appendChild(articleCard);
+//     });
+//     response.data.articles["technology"].forEach(item => {
+//       const articleCard = Cards(
+//         item.headline,
+//         item.authorPhoto,
+//         item.authorName
+//       );
+//       articleCards.appendChild(articleCard);
+//     });
+//     response.data.articles["jquery"].forEach(item => {
+//       const articleCard = Cards(
+//         item.headline,
+//         item.authorPhoto,
+//         item.authorName
+//       );
+//       articleCards.appendChild(articleCard);
+//     });
+//     response.data.articles["node"].forEach(item => {
+//       const articleCard = Cards(
+//         item.headline,
+//         item.authorPhoto,
+//         item.authorName
+//       );
+//       articleCards.appendChild(articleCard);
+//     });
+//   });
+
+// const articleCards = document.querySelector(".cards-container");
+
+function Cards(object) {
+  // Create element
+  const cardDiv = document.createElement("div");
+  const headlineDiv = document.createElement("div");
+  const authorDiv = document.createElement("div");
+  const imgDiv = document.createElement("div");
+  const imgInDiv = document.createElement("img");
+  const span = document.createElement("span");
+
+  // Class
+
+  cardDiv.classList.add("card");
+  headlineDiv.classList.add("headline");
+  authorDiv.classList.add("author");
+  imgDiv.classList.add("img-container");
+
+  // Content
+  headlineDiv.textContent = object.headline;
+  imgInDiv.src = object.authorPhoto;
+  span.textContent = object.authorName;
+
+  // Append
+  cardDiv.appendChild(headlineDiv);
+  cardDiv.appendChild(authorDiv);
+  authorDiv.appendChild(imgDiv);
+  authorDiv.appendChild(span);
+  imgDiv.appendChild(imgInDiv);
+
+  return cardDiv;
+}
+
+axios
+  .get("https://lambda-times-backend.herokuapp.com/articles")
+  .then(response => {
+    console.log(response);
+    response.data.articles["javascript"].forEach(item => {
```

Comment:
good hack. but what would you google to figure out how to loop through or alter an object to become an array